engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFutureImpl.java

```diff
@@ -152,7 +152,7 @@ public boolean isSuccessful() {
     }
 
     @Override
-    public boolean isErrornous() {
+    public boolean isErroneous() {
```

Comment:
(That of course also goes for the interface this implements... To lazy to hunt for it :))